docs/api/fibre_channel_initiator_aliases.md

```diff
@@ -0,0 +1,27 @@
+
+# nimbleclient.v1.api.fibre_channel_initiator_aliases
+
+
+## FibreChannelInitiatorAlias
+```python
+FibreChannelInitiatorAlias(self,
```

Comment:
Again same comment. Declaration should be one liner and consistent with other object sets.